javascript/linters/.eslintrc_react

```diff
@@ -3,6 +3,14 @@
   "plugins": [
     "react"
   ],
+  {
+    "settings": {
+      "react": {
+        "pragma": "React",  // Pragma to use, default to "React"
+        "version": "0.14" // React version, default to the latest React stable release
```

Comment:
Idem, I thought that it should be useful to keep them.
No problem if you want to remove comments and default values of course.